Svc/CmdSequencer/CmdSequencerImpl.cpp

```diff
@@ -60,9 +75,9 @@ namespace Svc {
 
     void CmdSequencerComponentImpl ::
       allocateBuffer(
-          const NATIVE_INT_TYPE identifier,
+          NATIVE_INT_TYPE identifier,
```

Comment:
I think these inputs should remain `const` here like they were before. This affects:

```
NATIVE_INT_TYPE identifier,
...
NATIVE_UINT_TYPE bytes
```

Should be:

```
const NATIVE_INT_TYPE identifier,
...
const NATIVE_UINT_TYPE bytes
```
```

Reply:
const updates added.